### PR TITLE
USHIFT-984: Enable presubmits for etcd/.* changes

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

File: ci-operator/config/openshift/microshift/openshift-microshift-main.yaml [hunk 1]
```diff
@@ -145,15 +145,15 @@ tests:
   run_if_changed: ^(rpm/*|packaging/.*|Makefile)
 - as: e2e-openshift-conformance-reduced
   cluster: build02
-  run_if_changed: ^(assets/.*|cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile|.*\.sh)
+  run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile|.*\.sh)
```
Comment (dhellmann) [inline ci-operator/config/openshift/microshift/openshift-microshift-main.yaml:148]: Would it make sense to use a pattern like `.*\.go` like we do for shell scripts? 

I see we aren't testing on changes to `validate-microshift`, either. I wonder if we change any of the files in other subdirectories often enough that we need this at all?

**Review by dhellmann — COMMENTED** [review]
This is an improvement, but maybe we can go further in another PR.

/lgtm

File: ci-operator/config/openshift/microshift/openshift-microshift-main.yaml (hunk 1 above)
Comment (pacevedom) [reply ci-operator/config/openshift/microshift/openshift-microshift-main.yaml:148]: We might need to rework this regex a little bit, will do it in a separate PR.

**Pull request merged**